SplunkPy: raising exception only if no incident to create (#20540)
demisto/content: Packs/SplunkPy/ReleaseNotes/2_4_6.md

#### Integrations
##### SplunkPy

>  - Fixed an issue where Notables was missed in case of error in the enrichment process.

Reviewer: ```suggestion
 - Fixed an issue where Notables were missed if an error occurred during the enrichment process.
```
